tests/test.py

```diff
@@ -484,9 +484,18 @@ def test_find_peak_in_breath(self):
         peak =find_peak_in_breath(sample_array,0,20)
         self.assertEqual(
             peak,
-            (9,13)
+            (9,13, 13)
         )
     
+    def test_find_peak_in_breath_convy(self):
+        sample_array= np.array(
+            [0,0,0,0,1,1,1,5,10,13,5,0,1,1,1,1,0,1,1,1,0,0,0]
+        )
+        peak =find_peak_in_breath(sample_array,0,20,'convy')
+        self.assertEqual(
+            peak,
+            (8,10, 11.5)
```

Comment:
Probably we would like to find index 9 and value 13? But with the smoothing everything is a bit different of course. It's a bit weird. But the test will tell us if the function is not failing

Reply:
Yes, the smoothing makes it not exact, but we have a signal that is at 2048, so this should be less of an issue